.github/workflows/cargo-update.yml

```diff
@@ -21,7 +25,8 @@ jobs:
         uses: peter-evans/create-pull-request@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: "build: Bump dependencies"
+          commit-message: "build: Bump Cargo.lock dependencies"
```

Comment:
Unrelated to this PR, but do we want to enforce this style of commit messages across all repositories now?

I personally haven't adopted it due to its verbosity/redundancy, but it seems like you and @maxrake have already used it for at least some (?) commits.

Reply:
It might still be premature to enforce [conventional commits](https://www.conventionalcommits.org) across all repositories. It is desirable IMO but the full use case is being worked out currently in the `phylum-dev/phylum-ci` repo...with lessons that may inform our decision going forward. I have been trying it out for a while now...with the help of the [`commitizen` tool](https://commitizen-tools.github.io/commitizen/) ...but it can also be done manually. My guess is that @kylewillmon is doing the same.

FYI...here is [a reference](https://github.com/commitizen/conventional-commit-types/blob/master/index.json) for what the various types mean.

Reply:
Correct, I've also just been trying it out. I'm not using any tool for it.